llvm/unittests/ExecutionEngine/Orc/JITLinkRedirectionManagerTest.cpp

```diff
@@ -81,8 +81,8 @@ TEST_F(JITLinkRedirectionManagerTest, BasicRedirectionOperation) {
                         JD->getDefaultResourceTracker(),
                         {{RedirectableSymbol, MakeTarget(initialTarget)}}),
                     Succeeded());
-  auto RTDef = cantFail(ES->lookup({JD}, RedirectableSymbol));
 
+  auto RTDef = cantFail(ES->lookup({JD}, RedirectableSymbol));
```

Comment:
Whitespace change?

Reply:
If there is, I can't see it? Looks ok in my editor too.